modules/RTC/TPCUtils.js

```diff
@@ -80,37 +158,103 @@ export class TPCUtils {
             }
         ];
+
+        if (this.codecSettings[codec].scalabilityModeEnabled) {
+            // Configure all 3 encodings when simulcast is requested through config.js for AV1 and VP9 and for H.264
+            // always since that is the only supported mode when DD header extension is negotiated for H.264.
+            if (this.codecSettings[codec].useSimulcast || codec === CodecMimeType.H264) {
+                for (const encoding of standardSimulcastEncodings) {
+                    encoding.scalabilityMode = VideoEncoderScalabilityMode.L1T3;
+                }
+
+                return standardSimulcastEncodings;
+            }
+
+            // Configure only one encoding for the SVC mode.
+            return [
+                {
+                    active: this.pc.videoTransferActive,
+                    maxBitrate: maxVideoBitrate,
+                    rid: SIM_LAYER_1_RID,
+                    scaleResolutionDownBy: HD_SCALE_FACTOR,
+                    scalabilityMode: this.codecSettings[codec].useL3T3
+                        ? VideoEncoderScalabilityMode.L3T3 : VideoEncoderScalabilityMode.L3T3_KEY
+                },
+                {
+                    active: false,
+                    maxBitrate: 0
+                },
+                {
+                    active: false,
+                    maxBitrate: 0
+                }
+            ];
+        }
+
+        return standardSimulcastEncodings;
+    }
+
+    /**
+     * Returns a boolean indicating whether the video encoder is running in full SVC mode, i.e., it sends only one
+     * video stream that has both temporal and spatial scalability.
+     *
+     * @param {CodecMimeType} codec
+     * @returns boolean
+     */
+    _isRunningInFullSvcMode(codec) {
+        return (codec === CodecMimeType.VP9 || codec === CodecMimeType.AV1)
+            && this.codecSettings[codec].scalabilityModeEnabled
+            && !this.codecSettings[codec].useSimulcast;
+    }
+
+    /**
+     * Returns a boolean indicating whether the bitrate needs to be capped for the local video track if it happens to
+     * be a screenshare track. The lower spatial layers for screensharing are disabled when low fps screensharing is in
+     * progress. Sending all three streams often results in the browser suspending the high resolution in low b/w and
+     * and low cpu conditions, especially on the low end machines. Suspending the low resolution streams ensures that
+     * the highest resolution stream is available always. Safari is an exception here since it does not send the
+     * desktop stream at all if only the high resolution stream is enabled.
+     *
+     * @param {JitsiLocalTrack} localVideoTrack - The local video track.
+     * @returns {boolean}
+     */
+    _isScreenshareBitrateCapped(localVideoTrack) {
+        return localVideoTrack.getVideoType() === VideoType.DESKTOP
+            && this.pc._capScreenshareBitrate
+            && !browser.isWebKitBased();
```

Comment:
Why not WebKit? Can you pl add a comment explaining?

Reply:
It has already been explained few lines above this :) https://github.com/jitsi/lib-jitsi-meet/pull/2350/files#diff-6cd0aeb8cb1acbd0118edb9d36b059e32f1984192a2c012bf5d6db68f02ab71aR248